File: codeforces/Way Too Long Words.js
var n = readline();

for (var w = 0; w < n; w++){
    var word = readline();

    if (word.length > 10){
        var long = word.length - 2;
        print(word[0] + String(long) + word[word.length - 1]);
    }else{
        print(word);
    }
}

File: codeforces/Way Too Long Words.py
#import sys
 
#input = sys.stdin.readline
 
n = int(input())
 
list1 = [str(input()) for _ in range(n)]
 
list2 = []
 
for i in list1:
    if len(i) > 10:
        i = i[0] + str(len(i) - 2) + i[-1]
        list2.append(i)
    else:
        list2.append(i)
        
for i in list2:
    print(i)
